fastlane/swift/LaneFileProtocol.swift

```diff
@@ -143,6 +143,9 @@ open class LaneFile: NSObject, LaneFileProtocol {
         // Call only on success.
         if !LaneFile.onErrorCalled.contains(lane) {
             fastfileInstance.afterAll(with: lane)
+        } else {
```

Comment:
could this also be written like this?

```
        if LaneFile.onErrorCalled.contains(lane) {
            log(message: "Error running lane: \(lane) 😥")
            fatalError("Aborting run due to lane failure")
        }
        fastfileInstance.afterAll(with: lane)
```

Reply:
I guess. But afterAll still won't be executed after a fatal error, of course.
Should i implement your suggestion?